tests/platform/test_sfp.py

```
@@ -56,9 +60,14 @@ def test_check_sfp_status_and_configure_sfp(testbed_devices, conn_graph_facts):
     * show interface transceiver eeprom
     * sfputil reset <interface name>
     """
-
     ans_host = testbed_devices["dut"]
 
+    loganalyzer = LogAnalyzer(ansible_host=ans_host, marker_prefix='sfp_cfg')
+    loganalyzer.load_common_config()
+
+    loganalyzer.ignore_regex.append("kernel.*Eeprom query failed*")
```

why are we ignoring errors from the eeprom instead of understanding why there are eeprom read failure? we should not have such errors in any case